public/js/publink.js

```diff
@@ -5,12 +5,33 @@ var valid_dataset = false;
 var last_retrieved = { dataset: "", publication: "" };
 
 $(document).ready(function() {
-    $('#retrieve_publication').button().click(function () {
-        retrievePublicationCitation();
+    jQuery.validator.addMethod("doiFormat", function(value, element) {
+        var regPattern = /^10.\d{4,9}\/[-._;()\/:A-Z0-9]+$/i
```

Comment:
This DOI regex is fairly strict. While the one on API is less strict. (https://github.com/griidc/pelagos/pull/502/files#diff-3462843700e70b69066a685d5c23b761R141)

Reply:
https://www.doi.org/doi_handbook/2_Numbering.html#2.2.1 Yeah, that regex is way too strict per the DOI specification 2.2.1, "There is no defined limit on the length of the DOI name, or of the DOI prefix or DOI suffix." Also, any printable character is valid. Also "The DOI name is case-insensitive and can incorporate any printable characters from the legal graphic characters of Unicode" so we've got expand allowable characters.

We talked about it once upon a time and agreed on 10.* It think

Reply:
This regex is suggested by Crossref. Maybe it should be a bit more strict here. But i'm ok with relaxing it to `10\..*`